client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -522,18 +538,52 @@ class EmailProvidersComparison extends Component {
 			'email-providers-comparison__keep-main-price': ! isEligibleForFreeTrial,
 		} );
 
-		const formattedPrice = translate( '{{price/}} /mailbox /month (billed monthly)', {
+		const titanIsMonthly = titanMailProduct && isTitanMonthlyProduct( titanMailProduct );
+
+		const titanMonthlyCost = titanIsMonthly
+			? titanMailProduct?.cost ?? 0
+			: ( titanMailProduct?.cost ?? 0 ) / 12;
+
+		const monthlyPrice = formatCurrency( titanMonthlyCost, currencyCode );
+
+		const priceComponents = {
 			components: {
-				price: (
-					<span className={ formattedPriceClassName }>
-						{ formatCurrency( titanMailProduct?.cost ?? 0, currencyCode ) }
-					</span>
-				),
+				price: <span className={ formattedPriceClassName }>{ monthlyPrice }</span>,
 			},
 			comment: '{{price/}} is the formatted price, e.g. $20',
-		} );
+		};
 
-		const discount = isEligibleForFreeTrial ? translate( '3 months free' ) : null;
+		const formattedPrice = titanIsMonthly
+			? translate( '{{price/}} /mailbox /month (billed monthly)', priceComponents )
+			: translate( '{{price/}} /mailbox /month (billed annually)', priceComponents );
+
+		const discount = ! isEligibleForFreeTrial ? null : (
+			<>
+				{ translate( '3 months free' ) }
+				{ ! titanIsMonthly && (
+					<span className="email-providers-comparison__discount-with-renewal">
+						<span>
+							{ ' ' }
+							{ translate(
+								'%(firstRenewalPrice)s/mailbox billed in 3 months, renews at %(standardPrice)s/mailbox',
+								{
+									args: {
+										firstRenewalPrice: formatCurrency(
+											( titanMailProduct?.cost ?? 0 ) * 0.75,
```

Comment:
While this works right now, it's a tad hard-coded. 

If the billing plan is switched, this becomes inaccurate. This can also happen if the introductory offer is eliminated from the billing plan.

We need a more robust way to fetch the prorated cost, and in the long run - the length of the free trial term.

Perhaps in another PR?

Reply:
I think we may need to enrich the data coming from the server to support this. But I am not sure how easy it would be to do that, mostly because I am not sure where we could store that data for new domains.